src/config/mongoose_config_spec.erl

```diff
@@ -1045,6 +1087,17 @@ fast_tls_defaults() ->
     #{ciphers => mongoose_tls:default_ciphers(),
       protocol_options => ["no_sslv2", "no_sslv3", "no_tlsv1", "no_tlsv1_1"]}.
 
+process_c2s_fast_tls(M = #{module := just_tls}) ->
+    M;
+process_c2s_fast_tls(M = #{cacertfile := _}) ->
+    M;
+process_c2s_fast_tls(#{verify_mode := peer}) ->
+    error(#{what => missing_cacertfile,
+            text => <<"You need to provide CA certificate (cacertfile) "
```

Comment:
Since this function call is placed at the root, the reported node will be 'root', and the user might not know where to put `verify_mode`. How about this?
```suggestion
            text => <<"You need to provide CA certificate (cacertfile) in the 's2s.listen' sections "
```

Reply:
Yes, good point.

Reply:
Yes, right, I meant the right function, but I somehow put the comment in the wrong place.

Reply:
:thumbsup: